repo/fsrepo/migrations/fetcher.go

```diff
@@ -0,0 +1,83 @@
+package migrations
+
+import (
+	"context"
+	"io"
+	"os"
+)
+
+const (
+	// Current dirstibution to fetch migrations from
+	CurrentIpfsDist = "/ipfs/Qme8pJhBidEUXRdpcWLGR2fkG5kdwVnaMh3kabjfP8zz7Y"
```

Comment:
TODO: Update before merging PR (this is out of date anyhow, but also we need to publish to dist with the new migration binaries anyway)

Reply:
Before merge, I will create a new dist with the latest migration binaries and change all references to this.